src/theme/DocItem/Layout/styles.module.scss

```
@@ -8,13 +30,31 @@
+  width: rem(15px);
+  height: rem(15px);
+  color: var(--ifm-color-content-secondary);
+  opacity: 0.7;
+}
+
+.versionBadge {
+  margin-bottom: rem(10px);
+  background-color: none;
+  border: none;
+  padding: 0;
+  background: none;
```

Stylelint (17.14.1)

[error] Overridden property "background-color" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)